esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -20,593 +20,602 @@ diagnostics:
 
   ### TIER 2 ##################################################################
 
-  ESACCI-AEROSOL:
-    description: ESACCI-AEROSOL
+  # ESACCI-AEROSOL:
```

Comment:
Please do not comment all the other datasets.
One can still test just PIOMAS using the command line option `--diagnositcs`.

Reply:
Sorry for that, I never wanted to commit that